renovate.json

```diff
@@ -1,5 +1,14 @@
 {
   "extends": [
     "cozy"
+  ],
+  "packageRules": [
+    {
+      "matchUpdateTypes": [
+        "patch"
+      ],
+      "matchCurrentVersion": ">= 1.0.0",
+      "automerge": true
```

Comment:
Does this means that renovate can automerge its PRs without our approval? I'm not sure I would be confident with that.

Reply:
I allowed this only for patches in semver (not minor updates, are minor updates rarely strictly follow semver and are often breaking). What do you think? It would help reduce bot spam